src/components/Form.js

```diff
@@ -29,7 +29,7 @@ const Form = (props) => {
                 value={userInput}
             />
             <button>Add</button>
-            {props.errorState ? <p>no data found</p> : null}
+            {props.errorState ? <p className='errP'>no data found</p> : null}
```

Comment:
To improve your code, you can remove props and use only the errorState. Example:
`const Form = (props) => `
to
`const Form = ({errorState, setUserSubmit})`